code/modules/food_and_drinks/food/snacks_bread.dm

```diff
@@ -35,6 +35,16 @@
 	customfoodfilling = 1
 	foodtype = GRAIN
 
+/obj/item/reagent_containers/food/snacks/store/bread/plain/teleporter
```

Comment:
At least give these a slightly modified icon

Reply:
I'll ask the local drop pod enthusiast if he wants to resprite a bread loaf and slices for this one.

Reply:
Or you could just codersprite on some green spots

Reply:
![image](https://user-images.githubusercontent.com/29008542/48668491-b074dc80-eabc-11e8-9cfa-1620824a034b.png)
denton has asked me to craft these, opinions?

Reply:
![image](https://user-images.githubusercontent.com/29008542/48668518-3abd4080-eabd-11e8-8d9e-a63a5be6f7b7.png)

Reply:
Pretty good.